admiral/pkg/clusters/envoyfilter.go

```diff
@@ -184,3 +186,8 @@ func getHosts(routingPolicy *v1.RoutingPolicy) string {
 	hosts = strings.TrimSuffix(hosts, ",")
 	return hostsKey + hosts
 }
+
+func getPlugin(routingPolicy *v1.RoutingPolicy) string {
+	plugin := routingPolicy.Spec.Plugin
```

Comment:
should we check for nil here?

Reply:
oops let me add that, its a check in quarter master so that should work